src/storage/test/PrefixBloomFilterBenchmark.cpp

```diff
@@ -0,0 +1,119 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "base/Base.h"
+#include <rocksdb/db.h>
+#include <folly/Benchmark.h>
+#include "fs/TempDir.h"
+#include "kvstore/RocksEngineConfig.h"
+#include "storage/test/TestUtils.h"
+#include "utils/NebulaKeyUtils.h"
+
+DEFINE_int64(part_number, 20, "partition numbers");
+DEFINE_int64(vertex_per_part, 100, "vertex count with each partition");
+
+namespace nebula {
+namespace storage {
+
+void mockData(kvstore::KVStore *kv) {
+    LOG(INFO) << "Prepare data...";
+    for (PartitionID partId = 0; partId < FLAGS_part_number; partId++) {
+        std::vector<kvstore::KV> data;
+        for (int32_t vertexId = partId * FLAGS_vertex_per_part;
+                vertexId < (partId + 1) * FLAGS_vertex_per_part; vertexId++) {
+            for (TagID tagId = 3001; tagId < 3010; tagId++) {
+                auto key = NebulaKeyUtils::vertexKey(partId, vertexId, tagId, 1);
+                auto val = folly::stringPrintf("%d_%d", vertexId, tagId);
+                data.emplace_back(std::move(key), std::move(val));
+            }
+        }
+        folly::Baton<true, std::atomic> baton;
+        kv->asyncMultiPut(0, partId, std::move(data),
+                [&](kvstore::ResultCode code) {
+                    baton.post();
+                    folly::doNotOptimizeAway(code);
+                });
+        baton.wait();
+        kv->flush(0);
+    }
+}
+
+void testPrefixSeek(kvstore::KVStore *kv, int32_t iters) {
+    for (decltype(iters) i = 0; i < iters; i++) {
+        for (PartitionID partId = 0; partId < FLAGS_part_number; partId++) {
+            for (int32_t vertexId = partId * FLAGS_vertex_per_part;
+                    vertexId < (partId + 1) * FLAGS_vertex_per_part; vertexId++) {
+                auto prefix = NebulaKeyUtils::vertexPrefix(partId, vertexId);
+                std::unique_ptr<kvstore::KVIterator> iter;
+                kv->prefix(0, partId, prefix, &iter);
+                iter->next();
```

Comment:
Would you post the benchmark without next? I am curious about how much we benefit from it.

Reply:
let me try

Reply:
![image](https://user-images.githubusercontent.com/47170471/90115840-3c499480-dd87-11ea-91f4-1a29c0084c5e.png)